Makefile

```diff
@@ -0,0 +1,14 @@
+help:			## Help command for displaying all other make commands
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+.PHONY:
+run-dev:		## Command for running a local dev instance of the flask application
+		FLASK_ENV=development poetry run flask run
+
+.PHONY:
+run-tests:		## Command for running all pytests
```

Comment:
nit: why do you have two commands for running tests? can you just have the `test` make command?

Reply:
I was thinking of it as like 2 options.  run-tests or if you forget that/prefer just test.  But maybe for alignment run-tests to match run-dev is better?

Reply:
`run-tests` makes the most sense to me; i prefer to just have a single command for things because then if you have to change stuff you're just changing it in the one place, and it removes the chance you'd forget to change the other as well